src/Microsoft.ML.Vision/ImageClassificationTrainer.cs

```diff
@@ -1490,26 +1490,22 @@ public void Score(in VBuffer<byte> image, Span<float> classProbabilities)
             return (ValueMapper<TSrc, TDst>)(Delegate)del;
         }
 
-        ~ImageClassificationModelParameters()
+        public void Dispose()
```

Comment:
>Dispose [](start = 20, length = 7)

I understand this is the right thing to do but who will call Dispose? Currently the framework does not support disposing transforms, in the past by not freeing the session we have seen memory leak in the scenario where a user creates multiple pipelines.

Reply:
`Session` and `Graph` both have finalizers.